meson.build

```diff
@@ -21,6 +20,9 @@ subproject('nix')
 # Docs
 subproject('internal-api-docs')
 subproject('external-api-docs')
+if not meson.is_cross_build()
```

Comment:
thought: Can't we bring a native build of nix into the derivation environment, using `buildPackages` and/or `nativeBuildInputs`? Also the dep has `native : true`, so that should work, I suppose. Can do later.